otherlibs/configurator/src/v1.mli

```diff
@@ -40,8 +40,8 @@ module C_define : sig
 
       {[
         # C.C_define.import c ~includes:"caml/config.h" ["ARCH_SIXTYFOUR",
-        Switch];; - (string * Configurator.C_define.Value.t) list =
-        ["ARCH_SIXTYFOUR", Switch true]
+                                                         Switch];; - (string * Configurator.C_define.Value.t) list =
```

Comment:
Seems unrelated to the PR. (I also have no idea whats going on here, but it seems like the original comment was mangled by a code styler, and now it happened again.)

Reply:
Hah. It's marked as code, so the Janestreet style restyled it as OCaml code.
I don't know why the non-Janestreet style restyled it so terribly, but I'm changing it to be "verbatim" instead of "code".